apps/sales/serializers.py

```
@@ -0,0 +1,69 @@
+                'product': product,
+                'quantity': item['quantity']
+            })
+
+        # 2. Call the Service
+        try:
+            return SaleService.create_sale(
+                merchant=merchant, 
+                sale_data=validated_data, 
+                items_data=processed_items
+            )
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)